mex/backend/graph/cypher/fetch_identities.cql

```diff
@@ -0,0 +1,38 @@
+<# Fetch only identity-related fields for the given set of filters.
+
+Globals:
+    extracted_labels: List of all extracted class labels
+    merged_labels: List of all merged class labels
+
+Args:
+    filter_by_had_primary_source: Whether the final query should filter by identifiers
+                                  of MergedPrimarySources referenced by hadPrimarySource
+    filter_by_identifier_in_primary_source: Whether the final query should filter by
+                                            the value of identifierInPrimarySource
+    filter_by_stable_target_id: Whether the final query should filter by stableTargetId
+
+Returns:
+    List of identity objects.
+-#>
+MATCH (n:<<extracted_labels|join("|")>>)-[:stableTargetId]->(merged:<<merged_labels|join("|")>>)
+MATCH (n:<<extracted_labels|join("|")>>)-[:hadPrimarySource]->(primary_source:MergedPrimarySource)
```

Comment:
granted, the second `:<<extracted_labels|join("|")>>` was redundant and thereby misleading, as the label filter has already been applied to the variable `n`. however, the second `MATCH` does not overwrite the first, it refines it. fixed in fd09bc3.

> MATCH can occur at the beginning of the query or later, possibly after a WITH. If it is the first clause, nothing will have been bound yet, and Neo4j will design a search to find the results matching the clause and any associated predicates specified in any WHERE part. This could involve a scan of the database, a search for nodes having a certain label, or a search of an index to find starting points for the pattern matching. Nodes and relationships found by this search are available as bound pattern elements, and can be used for pattern matching of paths. They can also be used in any further MATCH clauses, where Neo4j will use the known elements, and from there find further unknown elements. — https://neo4j.com/docs/cypher-manual/current/clauses/match